message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -2057,5 +2057,11 @@
       <field type="uint16_t[4]" name="rpm" units="rpm">RPM (eRPM).</field>
       <field type="uint16_t[4]" name="count">count of telemetry packets received (wraps at 65535).</field>
     </message>
+    <message id="11060" name="NAMED_VALUE_STRING">
+      <description>Send a key-value pair as string. The use of this message is discouraged for normal packets, but a quite efficient way for testing new messages and getting experimental debug output.</description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="char[10]" name="name" instance="true">Name of the debug variable</field>
```

Comment:
Maybe make it 16 to allow these debug values to potentially match up with parameters.

Reply:
> Maybe make it 16 to allow these debug values to potentially match up with parameters.

Could do - this just happens to be the same value as in `NAMED_VALUE_FLOAT`.  Hmmm.